src/js/components/DataTable/Body.js

```diff
@@ -85,67 +89,101 @@ const Body = forwardRef(
                 ? datumValue(datum, primaryProperty)
                 : undefined;
               const isSelected = selected && selected.includes(primaryValue);
+              const childExist = childExpand.includes(index);
+              const ischildExpanded = childExpand && childExist;
```

Comment:
Seems odd to test for `childExpand` here when we've already dereferenced it on the line above.

Reply:
Addressed this.